### ISSUE TITLE
there is a dent in an n

### ISSUE THREAD

**Issue by makc** [felixmariotto/three-mesh-ui#48]
at https://three-mesh-ui.herokuapp.com/#basic_setup
![Screen Shot 2020-11-23 at 18 44 10 ](https://user-images.githubusercontent.com/242577/99996431-02dbe880-2dbc-11eb-99b1-f8ed5c4ca78a.png)
![Screen Shot 2020-11-23 at 18 46 22 ](https://user-images.githubusercontent.com/242577/99996586-3e76b280-2dbc-11eb-8d13-36a007b01766.png)

**Comment by felixmariotto** [comment]
Hi @makc, I'm afraid I can't do much about this.

There is no MSDF texture generation in three-mesh-ui, I advise in my tutorials to go make one at [don macCurdy's generator](https://msdf-bmfont.donmccurdy.com). The [shader that parse the texture](https://github.com/felixmariotto/three-mesh-ui/blob/46f589a897a2526cf01df6f96b9c6ab6a9deb1fc/src/components/core/MaterialManager.js#L238-L247) in three-mesh-ui is very standard, I didn't invent anything here, and I would not know how to optimize it for this specific case.

**Comment by makc** [comment]
u is pretty much n upside down and there is no dent

**Comment by makc** [comment]
one of these has a buggy pixel
![Uploading Screen Shot 2020-11-23 at 21.03.54 .png…]()

**Comment by felixmariotto** [comment]
I tried to generate the same font again using the generator, and I get no dent but character position is slightly off. I will look into this more closely, I admit I didn't make a lot of test with other fonts than Roboto.

While digging in dom macCurdy's site files, I realised it's just a wrapper around [this](https://github.com/soimy/msdf-bmfont-xml), and it looks like it's possible to use it as a module. I will try and see if it's possible to use it client-side, it which case the user would just have to provide a TTF file, which would be awesome.
